stylesheets/components/badges/badges.css

```diff
@@ -0,0 +1,83 @@
+@import "/stylesheets/index.css";
+@import "/stylesheets/components/avatars/avatars.css";
+
+
+.general-badges {
+    margin-bottom: 0;
+}
+
+.badges-on-icons {
+    background: var(--badges-on-icons-section);
+}
+
+.icon-with-badge {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.badge-icon-showcase {
+    position: relative;
+}
+
+.badge {
+    background: var(--badge-default-bg);
+    height: 1.25rem;
+    width: 1.25rem;
+    border-radius: 50%;
+    color: var(--badge-default-text);
+    text-align: center;
+    position: absolute;
+    top: -5px;
+    right: -6px;
+}
+
+.icon-badge{
+    font-size: small;
+    padding: 1px;
+}
+.badges-on-avatars {
+    background: var(--badges-on-avatars-section);
+}
+
+.avatar-with-badge{
+    display: flex;
+    justify-content: space-evenly;
+    padding: 1rem;
+}
+
+.avatar-badge{
+    display: flex;
+    position: relative;
+}
+
+.av-badge{
+    right: 1px;
+    border: 2px solid var(--badge-border-default);
+}
+.av-sm-badge{
+    height: 10px;
+    width: 10px;
+    top: 1.4rem;  
+}
+
+.av-md-badge{
+    height: 1rem;
+    width: 1rem;
+    top: 2.1rem;
+}
+
+.av-lg-badge{
+    height: 1.25rem;
+    width: 1.25rem;
+    top: 3.25rem;
+}
+
+.grey-badge{
+    background: var(--grey-badge);
+}
+
+.green-badge {
+    background: var(--green-badge);
+}
+
```

Comment:
just a advise.... name the classes that are just changing color lil flexible...
bcoz u will have many component that will be of same color... (like green alert, green badge)
although it's just a advise u can always go as u please :

Reply:
Thanks, Ramandeep. But I have used different shades, hence different colors variables and classes. 
Will keep that in mind for the same ones!